#! /bin/sh /usr/share/dpatch/dpatch-run
## 14_disable-cmip.dpatch by  <fpeters@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Disable CMIP and GNM dissectors (CMIP doesn't build and GNM depends on
## DP: CMIP)

@DPATCH@
diff -urNad trunk~/epan/dissectors/Makefile.common trunk/epan/dissectors/Makefile.common
--- trunk~/epan/dissectors/Makefile.common	2008-02-27 20:58:31.000000000 +0100
+++ trunk/epan/dissectors/Makefile.common	2008-03-12 21:23:36.000000000 +0100
@@ -84,7 +84,6 @@
 	packet-ansi_tcap.c \
 	packet-camel.c \
 	packet-cdt.c \
-	packet-cmip.c \
 	packet-cmp.c \
 	packet-cms.c \
 	packet-crmf.c \
@@ -96,7 +95,6 @@
 	packet-erf.c \
 	packet-ess.c \
 	packet-ftam.c \
-	packet-gnm.c \
 	packet-gsm_map.c \
 	packet-h225.c \
 	packet-h235.c \
@@ -827,7 +825,6 @@
 	packet-cip.h	\
 	packet-clearcase.h	\
 	packet-clip.h	\
-	packet-cmip.h \
 	packet-cmp.h	\
 	packet-cms.h	\
 	packet-crmf.h	\
@@ -896,7 +893,6 @@
 	packet-frame.h	\
 	packet-ftam.h	\
 	packet-giop.h	\
-	packet-gnm.h	\
 	packet-gnutella.h	\
 	packet-gre.h	\
 	packet-gsm_a.h	\
diff -urNad trunk~/epan/dissectors/register.c trunk/epan/dissectors/register.c
--- trunk~/epan/dissectors/register.c	2008-02-27 21:00:50.000000000 +0100
+++ trunk/epan/dissectors/register.c	2008-03-12 21:25:32.000000000 +0100
@@ -124,7 +124,6 @@
   {extern void proto_register_clip (void); if(cb) (*cb)(RA_REGISTER, "proto_register_clip", client_data); proto_register_clip ();}
   {extern void proto_register_clnp (void); if(cb) (*cb)(RA_REGISTER, "proto_register_clnp", client_data); proto_register_clnp ();}
   {extern void proto_register_cltp (void); if(cb) (*cb)(RA_REGISTER, "proto_register_cltp", client_data); proto_register_cltp ();}
-  {extern void proto_register_cmip (void); if(cb) (*cb)(RA_REGISTER, "proto_register_cmip", client_data); proto_register_cmip ();}
   {extern void proto_register_cmp (void); if(cb) (*cb)(RA_REGISTER, "proto_register_cmp", client_data); proto_register_cmp ();}
   {extern void proto_register_cmpp (void); if(cb) (*cb)(RA_REGISTER, "proto_register_cmpp", client_data); proto_register_cmpp ();}
   {extern void proto_register_cms (void); if(cb) (*cb)(RA_REGISTER, "proto_register_cms", client_data); proto_register_cms ();}
@@ -268,7 +267,6 @@
   {extern void proto_register_gift (void); if(cb) (*cb)(RA_REGISTER, "proto_register_gift", client_data); proto_register_gift ();}
   {extern void proto_register_giop (void); if(cb) (*cb)(RA_REGISTER, "proto_register_giop", client_data); proto_register_giop ();}
   {extern void proto_register_gmrp (void); if(cb) (*cb)(RA_REGISTER, "proto_register_gmrp", client_data); proto_register_gmrp ();}
-  {extern void proto_register_gnm (void); if(cb) (*cb)(RA_REGISTER, "proto_register_gnm", client_data); proto_register_gnm ();}
   {extern void proto_register_gnutella (void); if(cb) (*cb)(RA_REGISTER, "proto_register_gnutella", client_data); proto_register_gnutella ();}
   {extern void proto_register_gprs_ns (void); if(cb) (*cb)(RA_REGISTER, "proto_register_gprs_ns", client_data); proto_register_gprs_ns ();}
   {extern void proto_register_gre (void); if(cb) (*cb)(RA_REGISTER, "proto_register_gre", client_data); proto_register_gre ();}
@@ -876,7 +874,6 @@
   {extern void proto_reg_handoff_clearcase (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_clearcase", client_data); proto_reg_handoff_clearcase ();}
   {extern void proto_reg_handoff_clip (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_clip", client_data); proto_reg_handoff_clip ();}
   {extern void proto_reg_handoff_clnp (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_clnp", client_data); proto_reg_handoff_clnp ();}
-  {extern void proto_reg_handoff_cmip (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_cmip", client_data); proto_reg_handoff_cmip ();}
   {extern void proto_reg_handoff_cmp (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_cmp", client_data); proto_reg_handoff_cmp ();}
   {extern void proto_reg_handoff_cmpp (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_cmpp", client_data); proto_reg_handoff_cmpp ();}
   {extern void proto_reg_handoff_cms (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_cms", client_data); proto_reg_handoff_cms ();}
@@ -1014,7 +1011,6 @@
   {extern void proto_reg_handoff_gift (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_gift", client_data); proto_reg_handoff_gift ();}
   {extern void proto_reg_handoff_giop (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_giop", client_data); proto_reg_handoff_giop ();}
   {extern void proto_reg_handoff_gmrp (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_gmrp", client_data); proto_reg_handoff_gmrp ();}
-  {extern void proto_reg_handoff_gnm (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_gnm", client_data); proto_reg_handoff_gnm ();}
   {extern void proto_reg_handoff_gnutella (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_gnutella", client_data); proto_reg_handoff_gnutella ();}
   {extern void proto_reg_handoff_gprs_ns (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_gprs_ns", client_data); proto_reg_handoff_gprs_ns ();}
   {extern void proto_reg_handoff_gre (void); if(cb) (*cb)(RA_HANDOFF, "proto_reg_handoff_gre", client_data); proto_reg_handoff_gre ();}
@@ -1484,6 +1480,6 @@
 
 gulong register_count(void)
 {
-  return 753 + 717;
+  return 751 + 717;
 
 }
